certbot-ci/certbot_integration_tests/utils/dns_server.py

```diff
@@ -30,7 +34,7 @@ class DNSServer:
     future to support parallelization (https://github.com/certbot/certbot/issues/8455).
     """
 
-    def __init__(self, unused_nodes, show_output=False):
+    def __init__(self, unused_nodes: Any, show_output: bool = False) -> None:
```

Comment:
Why `Any` and not `Sequence[str]`?

Reply:
Because "Implicit is better than explicit"? Or maybe it is the opposite? I am confused ;).

Anyway, fixed.